examples/with-svelte/apps/docs/package.json

```diff
@@ -3,6 +3,7 @@
 	"version": "0.0.1",
 	"private": true,
 	"type": "module",
+  "prepare": "svelte-kit sync",
```

Comment:
This is misplaced, needs to go into `scripts`

Reply:
@mcmxcdev fixed, thanks!